### DISCUSSION TITLE
Conso en euro ?

### DISCUSSION THREAD

**Discussion by BoD** [bokub/conso-api#29] [Aide] [answered]
Bonjour, je suppose que cette donnée n'est pas dispo chez Enedis, mais je préfère demander pour être sûr : les dépenses en Euro sont elles dispo ? Aujourd'hui j'ai cette info sur le site d'EDF, mais j'aimerais beaucoup une API - serait-ce possible ? Merci !

**Comment by bokub** [comment, marked as answer]
Hello !

Non, l'API ne donne aucune information sur ton fournisseur d'électricité, ton contrat, ou une quelconque information  qui n'est pas remontée par ton compteur. 

En gros enedis fournit l'électricité donc connaît ta consommation, mais ne s'occupe pas de gérer toute la partie contrat / facturation, qui est gérée par ton fournisseur.

**Reply by BoD** [reply]
Merci beaucoup pour la réponse. Je m'en doutais un peu mais cela confirme 👍
